src/nvim/buffer.c

```diff
@@ -94,6 +94,8 @@ static char *e_auabort = N_("E855: Autocommands caused command to abort");
 // Number of times free_buffer() was called.
 static int buf_free_count = 0;
 
+void buf_delete_signs(buf_T *buf, char_u *group);
```

Comment:
can remove, forward-declarations are auto-generated.

Reply:
I'm curious, how does that happen?

Reply:
done

Reply:
@bew we have a build step that runs Lua code, see `src/nvim/generators`.